options.html

```diff
@@ -109,7 +109,12 @@ <h3>Other</h3>
         <input id="controllerOpacity" type="text" value="">
       </div>
       <div class="row">
-        <label for="blacklist">Blacklisted sites on which extension is disabled<br/>(one per line)</label>
+        <label for="blacklist">Blacklisted sites on which extension is disabled<br/>
+            (one per line)<br/>
+            <br/>
+            <em>Regex is supported. Be sure it is in "//g" format.<br/>
+            ie: /(.+)youtube\.com(\/*)$/gi</em>
```

Comment:
Let's add a simple validator on the input here. If we allow bad expressions, the user will have a really hard time catching this error downstream.

Reply:
Makes sense, thanks for the feedback. I'll see what I can do.

Reply:
This is going to take a little more effort than I have time for tonight as I need to create a way to visually represent a failing regex and prevent submitting with a failure. I'll try and get something here within the next couple of days.

Reply:
Nevermind! I didn't notice before that there was already some signaling back to the user when they click "Save", so I just hijacked that. Seems like it's working https://github.com/ChadBailey/videospeed/commit/ac8b175a9870e56c7103df6f7637aa7b043baf66